frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx

```diff
@@ -20,6 +24,10 @@ export const ImageUploadField: React.FC<ImageUploadFieldProps> = (
   const [show, setShow] = React.useState(false);
   const [image, setImage] = React.useState();
 
+  let file: any = '';
+  if (image !== undefined) {
+    file = image;
```

Comment:
This is very ugly but it's the cleanest way I could make Typescript happy. Suggestions are welcome

Reply:
I have a better suggestion, you can define the type of image here.

```
interface Image {
  name?: string
}
```

then you can use `{image !== undefined ? (<h3>{image.name}</h3>): ''}`

Reply:
Ok, I added something based on that. Thank you!